exopite-simple-options/exopite-simple-options-framework-class.php

```diff
@@ -1107,7 +1248,7 @@ public function add_field( $field, $value = '' ) {
 
 			do_action( 'exopite_sof_after_generate_field', $field, $this->config );
 
-			echo apply_filters( 'exopite_sof_add_field', $output, $field, $this->config );
+			echo apply_filters( 'exopite_sof_add_field ', $output, $field, $this->config );
```

Comment:
'exopite_sof_add_field ' <- here is an extra space, can you please remove it? :)